Dockerfile

```diff
@@ -21,8 +21,9 @@ RUN apt-get update && \
 # Install gear python dependencies
 RUN pip3 install --upgrade pip && \
 	pip3.5 install \
-	flywheel_sdk==8.2.0 \
+	flywheel_sdk==10.7.0 \
```

Comment:
10.7.4 happened 3 days ago.  I keep this link handy: https://gitlab.com/flywheel-io/product/backend/core/-/tags